packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -530,6 +561,33 @@ void main() {
     }
   });
 
+  testWidgets('Scrollbar works with null Scroll Metrics', (WidgetTester tester) async {
+    final _TestScrollController scrollController = _TestScrollController();
+    await tester.pumpWidget(
+      Directionality(
+        textDirection: TextDirection.ltr,
+        child: MediaQuery(
+          data: const MediaQueryData(),
+          child: RawScrollbar(
+            isAlwaysShown: true,
+            controller: scrollController,
+            child: SingleChildScrollView(
+              controller: scrollController,
+              child: const SizedBox(width: 1000.0, height: 1000.0),
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await tester.pumpAndSettle();
+    expect(scrollController.offset, 0.0);
+    expect(
+        find.byType(RawScrollbar),
+        findsOneWidget,
+    );
```

Comment:
This does not check to see if anything has been painted, there should not be a scrollbar painted, correct?

Reply:
Yes, the metrics are null and can't be painted.

Reply:
It should probably check then that it `paintsNothing` to confirm that. :)